app/models/agents/data_output_agent.rb

```diff
@@ -156,6 +157,12 @@ def feed_description
       interpolated['template']['description'].presence || "A feed of Events received by the '#{name}' Huginn Agent"
     end
 
+    def xml_namespace
+      if (interpolated['itunes_compatible'] == "true")
+      	"xmlns:itunes=\"http://www.itunes.com/dtds/podcast-1.0.dtd\""
```

Comment:
You could also pull `xmlns:atom` and `xmlns:media` into this helper too.  For example:

```ruby
namespaces = ['xmlns:atom="http://www.w3.org/2005/Atom"', 'xmlns:media="http://search.yahoo.com/mrss/"']
if boolify(interpolated['itunes_compatible'])
  namespaces << 'xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"'
end
namespaces.join(' ')

```